Upstream clone libbpf into their build environment, but we want to use our
system libraries.

---
diff --git a/felix/bpf-gpl/Makefile b/felix/bpf-gpl/Makefile
index 8d89a4c..070637f 100644
--- a/felix/bpf-gpl/Makefile
+++ b/felix/bpf-gpl/Makefile
@@ -37,9 +37,7 @@ CFLAGS +=  \
 # distribution with the right kernel headers and recent libbpf package.
 # That's tricky because go-build is based on the upstream images
 # from the go team, and they don't provide anything newer.
-CFLAGS +=  \
-	-I ./libbpf/src/ \
-	-I ./libbpf/include/uapi
+CFLAGS += -I/usr/include/bpf
 
 # Workaround for Debian placing "asm/types.h" in /usr/include/x86_64-linux-gnu
 # We also pick up a couple of other definitions from here, such as the socket
